src/components/elementView/elementView.scss

```diff
@@ -86,9 +109,11 @@
 .bp3-tab-panel {
   overflow: auto;
   width: calc(100vw - 136pt);
+  font-weight: normal;
+  margin-left: 5pt;
 }
 .element-view {
-  width: 100%;
+  width: 95%;
```

Comment:
after playing around with the changes, it looks like the length of the new tabs (Decommissioned Assets and Offline Storage Sites being much longer than existing ones) made the right margin go away. this change seems to have fixed that